### DISCUSSION TITLE
Splitting Black Temple (and maybe SWP) in partial "Raid-Wing-Releases"

### DISCUSSION THREAD

**Discussion by Krateozshael** [chromiecraft/chromiecraft#7195] [Suggestions]
Hey there!

The staff have always put enormous pressure on themselves to release content of acceptable quality within a reasonable timeframe.  While well-meaning, this approach hasn't always proved to be successful, or popular among the playerbase.  Arguably, there are 2 major reasons for this:  limited manpower and the unnecessary self-imposed requirement that raids absolutely need to be released in full.

One potential solution is to release larger raids like Black Temple in stages, just as Blizzard originally did with ICC.  Instead of being content with a "70% state" why not release 3-4 bosses at a 90% state while continuing to work on those with more complicated scripting?  This would allow new brackets to be released on time, while satisfying the players' demand for new content.

Anyone who wonders why other parts of the raid aren't released at the same time can hop onto PTR to learn for themselves(and hopefully contribute by reporting the issues).  As long as those updates come regularly, the majority of players will feel that progress is being made and trust that the staff is doing their best.  At the same time, the devs will hopefully feel that they're working toward smaller, more achievable goals rather than feeling the burden of releasing an entire raid with unrealistic time constraints.

**Comment by heyitsbench** [comment]
Sunwell was also set up to have staggered boss releases in its original state, Black Temple the only point I can really think of that was gated from players was the Den of Mortal Delights, perhaps that could be a stagger point? Zul'Aman also pretty much only has a barred off point with Malacrass, but seeing as that only blocks off Malacrass and Zul'jin, I'm not sure how beneficial staggering that release would be.

**Reply by Krateozshael** [reply]
Well Zulaman as being a "catch up raid" more or less falls maybe out of that equation. but im mainly referring to BT with its 9 bosses and MAYBE SWP with its mechanic heavy fights.
Its a general idea that the Staff can look at imo.

**Reply by heyitsbench** [reply]
Well its status as a catch-up raid doesn't affect things IMO, this same idea of staggering boss releases was suggested a couple times for Karazhan bosses that I recall, only on the basis of its high boss count.

I don't know if I'm entirely supportive of this idea, but I'm certainly not against it.

**Comment by AcetylGer** [comment]
I think this is a great idea. If bugfixes are done fast the whole raid will be released in 3-4 weeks. If there is a lot of work to do then the realease can take more time with more testing done on the already released bosses.

**Comment by Cooties-cc** [comment]
Not fully sure this would expedite the bug fixing process, since it restricts _every_ player to a snails pace to test out content. To put more bluntly, there are some guilds/players that simply do more content than others in the same timeframes, thus more content overall is tested to be triaged. The pace in which issues are fixed isn't guaranteed either, as some bosses might take longer to fix than others or have fewer issues overall. This leads to also deciding to what degree does a portion of the raid need to be in working state (if not able to reach 90%) before the next part is released? At which point, if this does not exceed the pace of what a fully unlocked raid would achieve, what is the point of artificially restricting players? 

Using previous content chunks as a loose basis, some were broken up to lighten the loads of content delivery and pace of problems fixed. I don't think however the pace in which problems were corrected was much faster than if all the content was dropped at once. What it does do, is focus the direction of where the problems are fixed, thus this is where your idea of wing based releases of a raid could make a better partial experience. However it won't increase the rate of fixes done, since dev time is freely given and issues are not equal in difficulty. 

Overall I am for player freedom, which to me is allowing players to decide how much content they want to play/test. Not everyone reports issues, and it's the player(s) choice on how much content they want to play/test in their free time. The total bugs reported will be the same, albeit the focus would be spread. Restricting all players to an arbitrary pace, when they could have otherwise been given the option as a whole, doesn't seem like a good direction to me.

**Reply by Krateozshael** [reply]
Thank you for your comment, but this discussion is about helping the staff release higher quality content in a timely manner, given the damage done to the server's population and morale by putting out substandard releases in the past.  Players have always been able to test raids to their heart's content on PTR.

**Reply by Nyeriah** [reply]
> However it won't increase the rate of fixes done, since dev time is freely given and issues are not equal in difficulty.

Pretty much this. We have considered staggered raid releases before, however it doesn't really seem like it would help a whole lot. From a player perspective it is frustrating to only raid a few bosses, while from a dev perspective it doesn't speed things up much, considering issues have different levels of complexity and dev time/resources are scarce. It might actually slow things down as reports will also be staggered.

**Reply by Krateozshael** [reply]
Is there a possibility to put it on a server vote to actually see what "we" - the playerbase want? 
And not shrubbing it off.
There are voices for and against it and with the points i gave it sounds reasonable to atleast ask the community what they think of it.

**Comment by krutoj87** [comment]
Until WotLK I'm not going to raid much, so for TBC I don't really care, but for WotLK I don't like this approach. It just will make me get fed up ofc the first bosses faster. I'm in favor of releasing whole raids at once and giving the players the freedom to decide how much they want to play. Even though some bosses might still be bugged in some cases.

**Reply by Krateozshael** [reply]
We are only talking about TBC, because those raids have mostly to be fully rescripted and there is/was basically no work done on them. For WotLK alot of work is already done in AC, therefore this post isnt meant to address this, sorry for the confusion here.

**Reply by krutoj87** [reply]
Oh I didn't know. In this case just ignore my comment.
